ci-operator/config/openshift/deadmanssnitch-operator/openshift-deadmanssnitch-operator-master.yaml

```diff
@@ -1,30 +1,47 @@
-base_images:
-  base:
-    name: "4.4"
-    namespace: ocp
-    tag: base
-  openshift_release_golang-1.10:
-    name: release
-    namespace: openshift
-    tag: golang-1.10
-  openshift_release_golang-1.13:
-    name: release
-    namespace: openshift
-    tag: golang-1.13
 build_root:
-  image_stream_tag:
-    name: release
-    namespace: openshift
-    tag: golang-1.10
+  from_repository: true
 images:
 - dockerfile_path: build/Dockerfile
-  from: base
-  to: deadmanssnitch-operator
+  to: unused
```

Comment:
If this is going to `unused` then how is `src` determined?

Reply:
See above for where `src` comes from.

This stanza exists solely to test the docker build, so we don't end up with surprise failures after merge when the appsre pipeline runs. The resulting image isn't actually needed for any other presubmit testing, since all the other tests need to run directly on `src` -- i.e. "the boilerplate backing image, with the consumer repo cloned into it at the appropriate commit". 

And actually, the docker build needs to run from the same starting point. But we're already [keeping the Dockerfile in sync](https://github.com/openshift/deadmanssnitch-operator/blob/f9cfccd45c4f99b9768c2df8d44c96f2457b169b/build/Dockerfile#L1-L2) with what ends up in `src`. So we don't need to use `from src` here. (We _could_, but it actually complicates the process because it makes the pipeline mess with our Dockerfile in ways that turn out to be difficult to predict/manage consistently. It's also slower.)

Some day we may have integration/e2e tests that would need to deploy the built image to a real cluster. At that point, we would rename it appropriately and consume it from those (new) test cases.